src/it/resources/com/google/checkstyle/test/chapter4formatting/rule4861blockcommentstyle/InputFormattedBlockCommentStyle.java

```diff
@@ -0,0 +1,913 @@
+        break;
+      case "18":
+        break;
+      case "19":
+        // comment
+      case "20":
+        // comment
+      case "21":
+      default:
+        // comment
+        break;
+    }
+  }
+
+  private static void foo1() {
+    if (true) {
+      switch (1) {
+        case 0:
+
+        case 1:
+
+          // odd indentation comment
+          int b = 10;
+          break;
+        default:
+          // comment
+      }
+    }
+  }
+
+  /** some javadoc. */
+  public void fooDotInCaseBlock() {
+    int i = 0;
+    String s = "";
+
+    switch (i) {
+      case -2:
+        // what
+        i++;
+        break;
+      case 0:
+        // what
+        s.indexOf("ignore");
+        break;
+      case -1:
+        // what
+
+        s.indexOf("no way");
+        // odd indentation comment
+        break;
+      case 1:
+      case 2:
+        i--;
+        break;
+      case 3:
+        // fall through
+      default:
+    }
+
+    String breaks =
+        ""
+
+            // odd indentation comment
+            + "</table>"
+            // middle
+            + ""
+        // end
+        ;
+  }
+
+  /** some javadoc. */
+  public void foo2() {
+    int a = 1;
+    switch (a) {
+      case 1:
+      default:
+        // odd indentation comment
+    }
+  }
+
+  /** some javadoc. */
+  public void foo3() {
+    int a = 1;
+    switch (a) {
+      case 1:
+      default:
+
+        // comment
+    }
+  }
+
+  /** some javadoc. */
+  public void foo4() {
+    int a = 1;
+    switch (a) {
+      case 1:
+        int b;
+        // odd indentation comment
+        break;
+      default:
+    }
+  }
+
+  /** some javadoc. */
+  public void foo5() {
+    int a = 1;
+    switch (a) {
+      case 1:
+        int b;
+        // comment
+        break;
+      default:
+    }
+  }
+
+  /** some javadoc. */
+  public void foo6() {
+    int a = 1;
+    switch (a) {
+      case 1:
+        int b;
+        // comment
+        break;
+      default:
+    }
+  }
+
+  /** some javadoc. */
+  public void foo7() {
+    int a = 2;
+    String s = "";
+    switch (a) {
+        // comment
+        // comment
+        // comment
+      case 1:
+        // violation 2 lines above '.* indentation should be the same level as line 676.'
+      case 2:
+        // comment
+        // comment
+        foo1();
+        // comment
+        break;
+      case 3:
+        // comment
+        // comment
+        // comment
+
+      case 4:
+        // odd indentation comment
+      case 5:
+        s.toString().toString().toString();
+        // odd indentation comment
+        // odd indentation comment
+        // odd indentation comment
+        break;
+      default:
+    }
+  }
+
+  /** some javadoc. */
+  public void foo8() {
+    int a = 2;
+    String s = "";
+    switch (a) {
+        // comment
+        // comment
+        // comment
+      case 1:
+        // violation 2 lines above '.* indentation should be the same level as line 709.'
```

Comment:
I wrote:
>Please add violation comments above switch ("violation below x lines") to not interfere much inside switch blocks.

Please do:
```
// violation 4 lines below '.* indentation should be the same level as line 709.'
switch (a) {
```

To make it clear that violation comment doesn't contribute to violation by any means as it is out side of switch at all

Reply:
sorry my bad. Changed the comment location as you suggested.